src/components/RunTestsButton.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { action } from "@storybook/addon-actions";
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { TOOL_ID } from "../constants";
+import { RunTestsButton } from "./RunTestsButton";
+
+const meta = {
+  component: RunTestsButton,
+  args: {
+    isStarting: false,
+    projectId: "projectId",
+    accessToken: "accessToken",
```

Comment:
Can we pass in `isLoggedIn` or similar? Does the button need to know the accessToken?

Reply:
It could be! I used the prop directly, but renaming the prop and passing !!accessToken would be cleaner